### PR TITLE
gonzalezf fib.py

### PR DESCRIPTION
**Pull request by gonzalezf**
- [ ] I added myself to [`people.json`](./people.json)!

### CODE REVIEW

File: people.json [hunk 1]
```diff
@@ -180,12 +180,23 @@
     {
         "github": "wilbowma",
         "name": "William Bowman",
-        "title": "Assistant Professor of Computer Science, UBC",
+        "title": "MSc Computer Science Student, UBC",
```
Comment (braxtonhall) [inline people.json]: I assume this is a mistake!

File: people.json [hunk 2]
```diff
@@ -180,12 +180,23 @@
...
             {
                 "name": "fib-lang",
                 "link": "https://github.com/wilbowma/fib-lang/tree/2ec2d1dfd141220882d824cf3dac5b374ed291f3"
             }
         ]
     }
+    {
+        "github": "gonzalezf",
+        "name": "Felipe González-Pizarro",
+        "title": "Assistant Professor of Computer Science, UBC",
```
Comment (braxtonhall) [inline people.json]: As is this???

File: people.json [hunk 3]
```diff
@@ -180,12 +180,23 @@
...
-        "title": "Assistant Professor of Computer Science, UBC",
+        "title": "MSc Computer Science Student, UBC",
         "entries": [
             {
                 "name": "fib-lang",
                 "link": "https://github.com/wilbowma/fib-lang/tree/2ec2d1dfd141220882d824cf3dac5b374ed291f3"
             }
         ]
     }
+    {
```
Comment (braxtonhall) [inline people.json:191]: line 190 will need a comma or the JSON will break :(

**Review by braxtonhall — CHANGES_REQUESTED** [review]
JSON is dumb

File: people.json (hunk 1 above)
Comment (wilbowma) [reply people.json]: Dang doesn't even believe I have a PhD.

File: people.json (hunk 2 above)
Comment (gonzalezf) [reply people.json]: Oops, I went too far haha

File: people.json (hunk 1 above)
Comment (gonzalezf) [reply people.json]: Oops, Sorry!

Comment (wilbowma) [reply people.json]: Just trolling

**Pull request closed without merging**